packages/api/src/router/absence.ts

```diff
@@ -6,14 +7,17 @@ export const absenceRouter = createTRPCRouter({
   createAbsence: protectedProcedure
     .input(
       z.object({
-        participantId: z.number().nonnegative(),
         absenceDate: z.date(),
       }),
     )
     .mutation(async ({ input, ctx }) => {
+      const user = await ctx.db.user.findFirst({ where: { id: ctx.userId } });
```

Comment:
i think findUnique makes a bit more sense here, even if they effectively do the same thing